arches/app/models/tile.py

```diff
@@ -475,13 +475,13 @@ def save(self, *args, **kwargs):
                     transaction_id=transaction_id,
                 )
 
-            if index:
-                self.index()
-
             for tile in self.tiles:
                 tile.resourceinstance = self.resourceinstance
                 tile.parenttile = self
                 tile.save(*args, request=request, index=index, **kwargs)
```

Comment:
@chiatt i would pass index=false here as you are going to handle indexing afterwards by the looks of things

Reply:
Ah, good catch @aj-he. Fixed it. Thanks!